Vagrantfile

```diff
@@ -240,9 +240,9 @@ Vagrant.configure(2) do |config|
         machine.vm.provision :deploy_tendrl, type: :ansible do |ansible|
           ansible.limit = 'all'
           ansible.groups = {
-            'tendrl2-server' => ['tendrl2-server']
+            'tendrl2-servers' => ['tendrl2-server']
           }
-          ansible.playbook = 'ansible/tendrl2-server.yml'
+          ansible.playbook = 'ansible/prepare-tendrl2-server.yml'
```

Comment:
We already have a playbook at ansible/tendrl2-server.yml. Why do we want to make a new one?